starter-code/database.js

```diff
@@ -43,8 +43,8 @@ class Database {
         // Implement the query to insert a user
         // user is the document that we want to insert
         // remeber once it's finish to comment callback('Error inserting user');
-        
-        callback('Error inserting user');
+        this.database.collection(users).insertOne(user,callback);
+       // callback('Error inserting user');
```

Comment:
Tienes callback disponible como una función independiente, por lo que puedes ejecutarla con .insertOne().

this.database.collection(users).insertOne(user,callback('Usuario añadido'));